UnitsNet/UnitDefinitions/ThermalResistance.json

```diff
@@ -0,0 +1,68 @@
+{
+  "Name": "ThermalResistance",
+  "BaseUnit": "HourSquareFeetDegreeFahrenheitPerBtu",
```

Comment:
Are you sure this should be the base unit? Normally SI units are chose as base units, where possible, so I would expect something like `SquareMeterKelvinPerWatt` to be the base unit (but it currently does not exist here). The next best unit that currently exists is `SquareMeterKelvinPerKilowatt` maybe. What do you think?